.gitignore

```diff
@@ -1,3 +1,4 @@
+/.phpcs-cache
```

Comment:
I'd recommend using `./cache/phpcs.cache.json`, so you can use a single folder to store phpcs, phpunit and phpstan cache files.

Reply:
Instead of `./.cache/phpcs.cache.json` I've used the same solution as phpstan-core - `/tmp/cache` directory